bin/e2e-test-runner

```diff
@@ -1,16 +1,49 @@
 #!/bin/bash
 set -e
 
+
+export DEBUG=1
+export NO_RESTART_LOOP=1 
+export CYPRESS_BASE_URL=http://localhost:8080
+export OPT_OUT_CAPTURE=1
+export SECURE_COOKIES=0
+export PRIMARY_DB=clickhouse
+export SKIP_SERVICE_VERSION_REQUIREMENTS=1
+export KAFKA_URL=kafka://kafka:9092
+export CLICKHOUSE_DATABASE=posthog_test
+export TEST=1
```

Comment:
non-blocking as it's not a change from previous

Is it possible to run with TEST not set? It would be nice, given this is an e2e to run close to production config. Same for DEBUG.

Reply:
running DEBUG=0 locally is problematic due to ssl type stuff etc.

TEST=0 I tried, but I think the plugin server and kafka then user the wrong clickhouse database. Spent some time on figuring out why but couldn't. i'll add a comment